CMakePresets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "version": 7,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 27,
```

Comment:
CMake 3.19 should be sufficient: https://cmake.org/cmake/help/v3.19/manual/cmake-presets.7.html